dallinger/config.py

```diff
@@ -6,6 +6,7 @@
 import imp
 import logging
 import os
+from sets import Set
```

Comment:
I'm -1 on this, it adds compatibility with very old versions of Python but breaks it with Python3. What was the rationale here?